mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +490,27 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+let newQuoteBtn = document.querySelector("#newQuote");
+let autoPlayStatus = document.querySelector(".controls p");
+newQuoteBtn.addEventListener("click", retrieveQuote);
+function retrieveQuote() {
+  let quoteDisplay = document.querySelector(".blockquote > h1");
+  let authorDisplay = document.querySelector(".blockquote > h4");
+  let randomQuote = pickFromArray(quotes);
+  quoteDisplay.textContent = randomQuote.quote;
+  authorDisplay.textContent = `-${randomQuote.author}`;
+}
+let checkBoxToggle = document.querySelector("#checkbox");
+checkBoxToggle.addEventListener("change", (e) => {
+  autoPlayStatus.innerText = "auto-play: ON";
+  let autoQuotes = setInterval(() => {
+    if (checkBoxToggle.checked) {
+      retrieveQuote();
+    } else {
+      clearInterval(autoQuotes);
+      autoPlayStatus.innerText = "auto-play: off";
+    }
+  }, 60000);
+});
+window.onload = retrieveQuote;
```

Comment:
this is all good! i like your design very much as well.
and still can't hold couple of advises for further improvements :-D (they are really minor though)
- whatever `newQuote` in this project would be better to have as `nextQuote`. In software world it's more common to refer with "next" to the existing item, and `new` - as branch new generated or added by user.
- `retrieveQuote` would be better to rename into `setQuote`. This is also some sort of best practice in naming convention for functions. from function with name starting with `retrieve` we usually expect some value to be returned whenever we call it. In your implementation this function rather sets the new values for the existing DOM elements
- variable `autoQuotes` will have better sense if renamed to `timeId` or `quoteAutoUpdateTimerId` etc.